src/otx/core/model/entity/base.py

```diff
@@ -106,3 +109,62 @@ def map_class_names(src_classes: list[str], dst_classes: list[str]) -> list[int]
             else:
                 src2dst.append(-1)
         return src2dst
+
+
+class OVModel(OTXModel, Generic[T_OTXBatchDataEntity, T_OTXBatchPredEntity]):
```

Comment:
Why don't we just
```suggestion
class OVModel(OTXModel):
```

Reply:
I did it at first time, however, it leads to mypy issues, because of the types override mismatch with the base class. Adding Generic type helps. Other solution is to use #noqa